add missing include statement to fix error
"error: incomplete type ‘G4NistManager’ used in nested name specifier"
author: Kenneth Hoste (HPC-UGent)
--- gate_v8.1.p01/source/geometry/src/GateDetectorConstruction.cc.orig	2019-01-10 19:33:54.073187889 +0100
+++ gate_v8.1.p01/source/geometry/src/GateDetectorConstruction.cc	2019-01-10 19:34:10.673027454 +0100
@@ -30,6 +30,7 @@
 #include "G4SDManager.hh"
 #include "G4Material.hh"
 #include "G4Material.hh"
+#include "G4NistManager.hh"
 
 #ifdef GATE_USE_OPTICAL
 #include "GateSurfaceList.hh"
